Week2/nadin.js

```diff
@@ -0,0 +1,181 @@
+'use strict'
+
+//strings
+
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+console.log(myString);
+console.log(myString.length); 
+
+const space = " ";
+myString = 'hello' + space + 'this' + space + 'is' + space + 'a' + space + 'difficult' + space + 'to' + space + 'read' + space + 'sentence';
+console.log(myString);
+
+// Arrays
+
+let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+let favoriteMauroAnimals = favoriteAnimals.push('turtle');
+console.log(favoriteAnimals);
+favoriteAnimals.splice(1, 0, 'meerkta');
+console.log('i think the new value of the array is:,["blowfish", "meerkta", "capricorn", "giraffe", "turtle"]');
+console.log(favoriteAnimals);
+console.log('The array has a length of:', favoriteAnimals.length);
+const index = favoriteAnimals.indexOf('meerkta');
+favoriteAnimals.splice(index, 1);
+console.log('The item I am looking for is at index:1');
+
+//1
+
+function sumThreeNumber (num1, num2, num3) {
+  return num1+num2+num3;
+}
+const total = sumThreeNumber(1,2,3);
+console.log(total);
+
+//2
+function colorCar (color) {
+return 'a' + ' ' + color + ' ' + 'car';
```

Comment:
You can add the spaces inside the strings, `a ' + color + ' car`.

Reply:
That is easier! Thanks 👍